face/UnifiedFaceCollection/python/unified_face_collection.py

```diff
@@ -0,0 +1,346 @@
+                    "personId": person_id,
+                    "personPersistedFaceId": person_persisted_face_id
+                }
+                user_data_face_list_json = json.dumps(user_data_face_list)
+
+                update_url_face_list = f"{self.face_api_url}/largefacelists/{self.large_face_list_id}/persistedfaces/{persisted_face_id}"
+                update_body_face_list = {
+                    "userData": user_data_face_list_json
+                }
+                update_response_face_list = requests.patch(update_url_face_list, headers=self.headers, json=update_body_face_list)
+                if update_response_face_list.status_code != 200:
+                    print("Failed to update userData for the face in the Large Face List.")
+                    return []
+
+                # Update the userData field with the mapping in the Large Person Group
+                user_data_large_person_group = {
+                    "persistedFaceId": persisted_face_id
+                }
+                user_data_large_person_group_json = json.dumps(user_data_large_person_group)
+
+                update_url_large_person_group = f"{self.face_api_url}/largepersongroups/{self.large_person_group_id}/persons/{person_id}/persistedfaces/{person_persisted_face_id}"
+                update_body_large_person_group = {
+                    "userData": user_data_large_person_group_json
+                }
+                update_response_large_person_group = requests.patch(update_url_large_person_group, headers=self.headers, json=update_body_large_person_group)
+                if update_response_large_person_group.status_code != 200:
+                    print("Failed to update userData for the face in the Large Person Group.")
+                    return []
+
+                return {
+                    "face_list": {
+                        "persistedFaceId": persisted_face_id,
+                    },
+                    "person_group": {
+                        "personId": person_id,
+                    }
+                }
+            else:
+                return []
+        
+        return {"face_list": { "persistedFaceId": face_list_result['persistedFaceId'] }}
+
+    def remove_face(self, persisted_face_id):
+        face_list_url = f"{self.face_api_url}/largefacelists/{self.large_face_list_id}/persistedfaces/{persisted_face_id}"
+        face_list_response = requests.get(face_list_url, headers=self.headers)
+        if face_list_response.status_code != 200:
+            return False
+
+        face_data = face_list_response.json()
+        user_data = face_data.get("userData")
+        if not user_data:
+            return False
+
+        user_data_dict = json.loads(user_data)
+        person_id = user_data_dict.get("personId")
+        person_persisted_face_id = user_data_dict.get("personPersistedFaceId")
+
+        if person_id and person_persisted_face_id:
+            # Remove face from Large Person Group
```

Comment:
If there's no face record linked to the person, should we also delete the person itself to make things clean?